web_m2x_options/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+
+{
+    "name": 'web_m2x_options',
+    "version": "12.0.1.0.0",
+    'category': 'Web',
+    "author": "ACSONE SA/NV, "
+              "0k.io, "
+              "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    'website': 'https://github.com/OCA/web',
+    'license': 'AGPL-3',
+    "depends": [
+        'base',
```

Comment:
@gustitammam remove it

Reply:
> @gustitammam remove it

@nikul-serpentcs Sorry, I'm not sure which line you mentioning, did you suggest to remove `base` as the dependency ?

Reply:
Yes, `base`
because it depends on `web`.

Reply:
> Yes, `base`
> because it depends on `web`.

@nikul-serpentcs `base` removed: https://github.com/OCA/web/pull/1086/commits/89aaf7342ac3851e34563fcc121a209c91896000